FAQ.md

```diff
@@ -146,6 +147,13 @@ Any change to a MachineDeployment will result in an immediate removal and replac
 
 # Cluster Autoscaler
 
+## What is the difference between the ClusterAutoscaler and MachineAutoscaler resources?
```

Comment:
we need to add a reference once we merge https://github.com/openshift/enhancements/pull/332

Reply:
a reference to the enhancement doc?